src/components/list/demoListControls/index.html

```diff
@@ -0,0 +1,29 @@
+<md-list ng-controller="ListCtrl">
+  <md-subheader class="md-no-sticky">Single Action Checkboxes</md-subheader>
+  <md-item ng-repeat="topping in toppings">
+    <p> {{ topping.name }} </p>
+    <md-checkbox class="md-secondary" ng-model="topping.wanted" aria-label="{{ topping.name }}">
```

Comment:
I'm actually wondering if the right-aligned single-action checkbox is even a thing. After looking at the spec, I don't see that combination. We could use the existing checkbox component if we make single-line items left aligned, they they show in the spec:
```html
<md-item ng-repeat="topping in toppings">
  <md-checkbox class="md-secondary" ng-model="topping.wanted">
    {{ topping.name }}
  </md-checkbox>
</md-item>
```
That also should NOT be wrapped in `<div role=button>`. That makes the checkbox unusable.